apps/users/models.py

```diff
@@ -361,7 +347,12 @@ def source(self):
 
     @property
     def name(self):
```

Comment:
I'm not sure why. I would guess they were both used slightly differently at one point and then they became the same thing. It shouldn't be too hard to remove `welcome_name` though.